CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and releases in NEOSDiscovery project adheres to [Semantic Versioning](http://se
 ### Changed
 - log level from debug to warn [PR#412](https://github.com/ualbertalib/NEOSDiscovery/pull/412)
 
+## [1.0.63] - 2020-09-21
```

Comment:
There's a "Changed" item in the unreleased section that should go with these ones too.